bots/censussync/src/main/java/org/openjdk/skara/bots/censussync/CensusSyncSplitBot.java

```diff
@@ -0,0 +1,254 @@
+        var dir = to.resolve("groups");
+        var ret = new ArrayList<Path>();
+        for (var group : XML.children(census, "group")) {
+            Files.createDirectories(dir);
+
+            String lead = null;
+            var members = new ArrayList<String>();
+            for (var person : XML.children(group, "person")) {
+                if (XML.hasAttribute(person, "role")) {
+                    var role = XML.attribute(person, "role");
+                    if (!role.equals("lead")) {
+                        throw new IOException("Unexpected role: " + role);
+                    }
+                    lead = XML.attribute(person, "ref");
+                } else {
+                    members.add(XML.attribute(person, "ref"));
+                }
+            }
+
+            var name = XML.attribute(group, "name");
+            var fullName = XML.child(group, "full-name").getTextContent();
+            var filename = dir.resolve(name + ".xml");
+            try (var file = newPrintWriter(filename)) {
+                file.format("<?xml version=\"1.0\" encoding=\"UTF-8\" ?>%n");
+                file.format("<group name=\"%s\" full-name=\"%s\">%n", name, escape(fullName));
+                file.format("    <lead username=\"%s\" />%n", lead);
+                for (var member : members) {
+                    file.format("    <member username=\"%s\" />%n", member);
+                }
+
+                file.format("</group>%n");
+            }
+            ret.add(filename);
+        }
+        return ret;
+    }
+
+    private static List<Path> syncProjects(Element census, Path to) throws IOException {
+        var dir = to.resolve("projects");
+        var ret = new ArrayList<Path>();
+        for (var project : XML.children(census, "project")) {
+            Files.createDirectories(dir);
+
+            String lead = null;
+            var committers = new ArrayList<String>();
+            var reviewers = new ArrayList<String>();
+            var authors = new ArrayList<String>();
+
+            var name = XML.attribute(project, "name");
+
+            for (var person : XML.children(project, "person")) {
+                var role = XML.attribute(person, "role");
+                var username = XML.attribute(person, "ref");
+                switch (role) {
+                    case "lead":
+                        lead = username;
+                        break;
+                    case "reviewer":
+                        reviewers.add(username);
+                        break;
+                    case "committer":
+                        committers.add(username);
+                        break;
+                    case "author":
+                        authors.add(username);
+                        break;
+                    default:
+                        if (name.equals("openjfx") && (username.equals("dwookey") || username.equals("jpereda"))) {
```

Comment:
I know this is already integrated, but these sounds like weird hard-coded exceptions? If there are issues in the census data, maybe they should be fixed instead?